[codemod] Fix some namespace issues in caffe2

r-barnes opened the pull request:
Summary:
Removes `using namespace` from a header file. Having `using namespace` in a header file is *always* a bad idea. A previous raft of diffs provided appropriate qualifications to everything that relied on this `using namespace`, so it is now safe to remove it in this separate diff.

Helps us enable `-Wheader-hygiene`.

Test Plan: Sandcastle

Reviewed By: dmm-fb

Differential Revision: D54838298




cc @gujinghui @PenghuiCheng @XiaobingSuper @jianyuh @jgong5 @mingfeima @sanchitintel @ashokei @jingxu10 @min-jean-cho @yanbing-j @Guobing-Chen @Xia-Weiwen

malfet reviewed (commented):
Haven't you forgot to remove `using namespace caffe2` from the header?

malfet reviewed (changes requested):
Please actually remove `using namespace caffe2 from `conv_transpoe_unpool...h`

r-barnes committed:
[codemod] Remove `using namespace` from header in caffe2/caffe2/ideep/operators/conv_transpose_unpool_base_op.h (#121847)

Summary:

Removes `using namespace` from a header file. Having `using namespace` in a header file is *always* a bad idea. A previous raft of diffs provided appropriate qualifications to everything that relied on this `using namespace`, so it is now safe to remove it in this separate diff.

Helps us enable `-Wheader-hygiene`.

Test Plan: Sandcastle

Differential Revision: D54838298

clee2000 commented:
@pytorchbot merge -f "internal diff was landed"
